### ISSUE TITLE
JAWS does not read out group labels in an embedded browser

### ISSUE THREAD

**Issue by dp21g** [FreedomScientific/standards-support#802]
### Summary

In an embedded edge browser, JAWS 2023 does not announce the group it belongs to.(this applies to using fieldset and legend or the role of group.)

> Steps to create a local embedded browser here: https://github.com/dp21g/embedded-browser

Within an embedded browser:
1. Go to https://www.w3.org/WAI/ARIA/apg/patterns/checkbox/examples/checkbox/
2. Tab over to "Lettuce"
3. It reads "Letuce checbox not checked"

### Expected result

It should read "Sandwich Condiments, Lettuce checkbox not checked".
However in 2019.1912.1 version of JAWS it does read the "Sandwich Condiments, Lettuce checkbox not checked"  

### Actual result

It reads "Letuce checbox not checked"

#### Operating System and version
Windows

#### Browser and version:
Version 120.0.2210.133 (Official build) (64-bit)

**Comment by stevefaulkner** [comment]
assigned to @BrettLewisVispero
